generators/csharp/sdk/src/readme/ReadmeSnippetBuilder.ts

```diff
@@ -121,6 +134,58 @@ try {
         return [];
     }
 
+    private buildForwardCompatibleEnumSnippets(): string[] {
+        const enumsWithValues = Object.values(this.context.ir.types).filter((t) => {
+            if (t.shape.type !== "enum") {
+                return false;
+            }
+            if (t.shape.values == null || t.shape.values.length === 0) {
+                return false;
+            }
+            return true;
+        });
+        if (enumsWithValues.length === 0) {
+            return [];
+        }
+        const firstEnum = enumsWithValues[0] as TypeDeclaration & {
+            shape: Type.Enum;
+        };
+
+        const enumName = firstEnum.name.name.pascalCase.safeName;
+        const enumCamelCaseName = firstEnum.name.name.camelCase.safeName;
+        const enumNamespace = this.context.getNamespaceFromFernFilepath(firstEnum.name.fernFilepath);
+        const firstEnumValue = firstEnum.shape.values[0] as EnumValue;
+        const firstEnumValueName = firstEnumValue.name.name.pascalCase.safeName;
+        const firstEnumValueWire = firstEnumValue.name.wireValue;
+
+        return [
+            this.writeCode(`
+using ${enumNamespace};
+
+// Using a built-in value
+var ${enumCamelCaseName} = ${enumName}.${firstEnumValueName};
+
+// Using a custom value
+var custom${enumName} = ${enumName}.FromCustom("custom-value");
+
+// Using in a switch statement
+switch (${enumCamelCaseName}.Value)
+{
+    case ${enumName}.Values.${firstEnumValueName}:
+        Console.WriteLine("${firstEnumValueName}");
+        break;
+    default:
+        Console.WriteLine($"Unknown value: {${enumCamelCaseName}.Value}");
```

Comment:
Again, hard coded names like `Value` aren't going to be guaranteed - but wait for the generic fix coming soon.